### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by Phil-Ah** [angular-dashboard-framework/angular-dashboard-framework#336]
Some elements of the interface are not accessible to screenreader. The `<a>` tags without href could be replaced with buttons.
